readme.md

```diff
@@ -0,0 +1,77 @@
+**Copyright 2021 Google Inc.**  
+**Licensed under the Apache License, Version 2.0 (the "License");**  
+**you may not use this file except in compliance with the License.**  
+**You may obtain a copy of the License at**  
+
+**http://www.apache.org/licenses/LICENSE-2.0**  
+
+**Unless required by applicable law or agreed to in writing, software**  
+**distributed under the License is distributed on an "AS IS" BASIS,**  
+**WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.**  
+**Author: Muhammad Zain ul Islam, PSO Google.**  
+**Your use of it is subject to your agreement with Google.**  
+**Following open source material have been used while writing the code as well**  
+**https://www.postgresqltutorial.com/postgresql-python/connect/**  
+**https://www.psycopg.org/docs/advanced.html#replication-support**  
+
+#Description and Instructions for running the code to deliver every change—insert, update, and delete  data from CloudSql Postgres into Pub/Sub.
+**Currently the solution will be deployed on a VM in GCP and will be connected to Postgres by a replication slot and each change will be pubslished to Pub/sub**
+
+**Programming Language:**  
+Python 3.7+
+
+**Steps to configure the Environment**
+1) Create a new VM or use an existing one. 
+2) While creating VM, allow service account access to Google APIs.
+3) Assign the service account IAM permission through IAM to be able to publish events to Pub/sub.
+
+4. Install pip3 for python3.
```

Comment:
On the VM or locally to deploy the solution? 
If on the VM, How to ssh into the VM?
Should we use a Python virtual env?